src/icalendar/parser.py

```diff
@@ -206,10 +206,57 @@ def fdel(self: Parameters):
 
 
 class Parameters(CaselessDict):
-    """Parser and generator of Property parameter strings. It knows nothing of
-    datatypes. Its main concern is textual structure.
+    """Parser and generator of Property parameter strings.
+
+    It knows nothing of datatypes.
+    Its main concern is textual structure.
+
+    Examples:
+
+        Modify parameters:
+
+        .. code-block:: pycon
+
+            >>> from icalendar import Parameters
+            >>> params = Parameters()
+            >>> params['VALUE'] = 'TEXT'
+            >>> params.value
+            'TEXT'
+            >>> params
+            Parameters({'VALUE': 'TEXT'})
+
+        Create new parameters:
+
+        .. code-block:: pycon
+
+            >>> params = Parameters(value="BINARY")
+            >>> params.value
+            'BINARY'
+
+        Set a default:
+
+       .. code-block:: pycon
```

Comment:
Funky format. See third example in https://icalendar--941.org.readthedocs.build/en/941/reference/api/icalendar.parser.html#icalendar.parser.Parameters

```suggestion
        .. code-block:: pycon
```